models/classes/class.Parser.php

```diff
@@ -20,114 +20,61 @@
  *               2009-2012 (update and modification) Public Research Centre Henri Tudor (under the project TAO-SUSTAIN & TAO-DEV);
  *
  */
+
+declare(strict_types=1);
+
+use oat\oatbox\filesystem\File;
+use oat\tao\model\ItemDomValidation;
 use oat\oatbox\service\ServiceManager;
 use oat\tao\model\upload\UploadService;
 
 /**
  * The Parser enables you to load, parse and validate xml content from an xml
  * Usually used for to load and validate the itemContent  property.
  *
- * @access public
  * @author Bertrand Chevrier, <bertrand.chevrier@tudor.lu>
- * @package tao
-
  */
 class tao_models_classes_Parser
 {
+    public const SOURCE_FILE = 1;
+
+    public const SOURCE_URL = 2;
+
+    public const SOURCE_STRING = 3;
+
+    // Current file is \oat\oatbox\filesystem\File object
+    public const SOURCE_FLYFILE = 4;
```

Comment:
```suggestion
    public const SOURCE_FILE = 1;
    public const SOURCE_URL = 2;
    public const SOURCE_STRING = 3;
    
    /**
     * Current file is \oat\oatbox\filesystem\File object
     */
    public const SOURCE_FLYFILE = 4;
```

Thanks for adding constant visibility. No line break is required unless there is a comment